blog.css

```diff
@@ -0,0 +1,313 @@
+    height: 500px;
+    width: 100vw;
+    background-color: lightgray;
+    margin-top:200px;
+    
+}
+
+.blogcomment{
+    display: flex;
+    font-size: 28px;
+    flex-direction: column;
+    font-weight: 450;
+    font-family: 'Peddana', serif;
+    color: black;
+    position: relative;
+    margin-left: 500px;
+    margin-top:95px;
+}
+
+#bold{
+    font:bolder;
+    font-weight: 900;
+    font-size: 26px;
+    position: relative;
+    display: inline;
+    justify-self: auto;
+    margin-top:100px;
+}
+p20{
+    display: flex;
+    position: relative;
+    font-size: 18px;
+    justify-content: center;
+    flex-direction: column;
+    font-weight: 700;
+    font-family: 'Peddana', serif;
+    color: black;
+    max-width: 500px;
+    margin-top: 180px;
+    right: 250px;
+}
+
+.text{
+    display: flex;
+    position: relative;
+    font-size: 25px;
+    font-family:'Peddana', serif;
+    top: -550px;
+    left:-650px;
+  
+}
+
+.text1{
+    display: flex;
+    position: relative;
+    font-size: 25px;
+    font-family:'Peddana', serif;
+    top: -570px;
+    left:-480px;
+}
+
+.msg{
+    display:flex;
+    position: relative;
+    top:-550px;
+    left:-650px;
+}
+
+.media{
+    height: 30px;
+    width:30px;
+    position: relative;
+    right:250px;
+    top: 20px;
+}
+.media1{
+    height:30px;
+    width:30px;
+    position: relative;
+    right:250px;
+    top:9px;
+
+}
+.media2{
+    height:30px;
+    width:30px;
+    position: relative;
+    right:250px;
+    top:5px;
+
+}
+
+.media3{
+    height:30px;
+    width:30px;
+    position: relative;
+    bottom:190px;
+}
+
+.media4{
+    height:30px;
+    width:30px;
+    position: relative;
+    bottom:210px;
+}
+
+.media5{
+    height:30px;
+    width:30px;
+    position: relative;
+    bottom:215px;
+}
+.media6{
+    height:30px;
+    width:30px;
+    position: relative;
+    bottom:220px;
+}
+
+#drib{
+position: relative;
+font-size: 10px;
+right:210px;
+bottom:15px;
+}
+#mail{
+    position: relative;
+    font-size: 10px;
+    right:210px;
+    bottom:20px;
+
+}
+#fb{
+    position: relative;
+    font-size: 10px;
+    right:210px;
+    bottom:25px;
+}
+#address{
+    position: relative;
+    font-size: 10px;
+    left:70px;
+    bottom:240px;
+}
+#IG{
+    position: relative;
+    font-size: 10px;
+    left:70px;
+    bottom:260px;
+}
+#num{
+    position: relative;
+    font-size: 10px;
+    left:70px;
+bottom:260px;
+}
+#twitt{
+    position: relative;
+    font-size: 10px;
+    left:70px;
+    bottom:260px;
+}
+
+#sendmessage {
+    display: flex;
+    justify-content: center;
+    align-content: flex-start;
+    font-family: 'Peddana', serif;
+    color:white;
+    font-size: 16px;
+    font-weight: 600;
+    background-color:darkorange;
+    height: max-content;
+    width: 160px;
+    padding:5px 5px;
+    position:relative;
+    top: 600px;
+    left:285px;
+
+}
+
+#sendmessage:hover{
+background-color:turquoise;
+border-radius: 50%;
+left:450px;
+}
+
```

Comment:
This on hover is causing your button to jump around on the scree, I believe it has to do with your `left:450px` property.